packages/base/src/index.mjs

```diff
@@ -417,7 +417,20 @@ const rules = createConfig({
     ],
     'jsdoc/valid-types': 'error',
 
-    // 'promise/no-multiple-resolved': 'error',
+    /* promise plugin rules */
+    'promise/catch-or-return': [
+      'error',
+      {
+        allowFinally: true,
+      },
+    ],
+    'promise/param-names': [
+      'error',
+      {
+        resolvePattern: '^_?resolve',
+        rejectPattern: '^_?reject',
+      },
+    ],
```

Comment:
`resolvePattern` and `rejectPattern` are set to `^_?resolve$` and `^_?reject$` by default respectively, meaning that the parameters passed to a `Promise` constructor function must be named exactly `resolve`, `reject`, or `_resolve`, `_reject`.

This change allows for an optional suffix after `resolve` and `reject`, for example `resolvePromise` or `rejectPromise`, which can be useful when promises are nested.